quiz.html

```diff
@@ -42,10 +42,10 @@ <h1>Skin Type Quiz</h1>
 						<h1>How does your skin typically feel in the morning?</h1>
 						<h2>1 / 5</h2>
 						<div class="choices">
-							<div class="choicebox"><img src="images/oily.png"><p><input type="radio" name="question1" value="Oily">Oily</p></div>
-							<div class="choicebox"><img src="images/dry.png"><p><input type="radio" name="question1" value="Dry">Dry</p></div>
-							<div class="choicebox"><img src="images/normal.png"><p><input type="radio" name="question1" value="Normal">Normal</p></div>
-							<div class="choicebox"><img src="images/sensitive.png"><p><input type="radio" name="question1" value="Sensitive">Sensitive</p></div>
+							<div class="choicebox"><img src="images/quiz/oily.png"><p><input type="radio" name="question1" value="Oily">Oily</p></div>

```

Comment:
Organised my folders a little better on my local storage so that eventually all images will not get jumbled in one "Images" folder.